binancial/data/get_klines_realtime.py

```
@@ -48,30 +71,57 @@ def handle_socket_message(msg):
+            
         except Exception as error:
-
-            print("MSG : " + str(msg))
-            print("Exception : " + str(error))
-
-            stream.stop_socket(stream)
-
-    stream.start_kline_socket(callback=handle_socket_message,
-                              symbol=symbol,
-                              interval='1m')
+            logger.error(f"Error processing message: {error}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)